### ISSUE TITLE
Black screen on ATI/AMD video card

### ISSUE THREAD

**Issue by Lithium64** [gonetz/GLideN64#479]
I have a ATI/AMD HD 4650 1GB that supports OpenGL 3.3 and DirectX 10.1, but I just get a black screen when I run the emulator.

![2](https://cloud.githubusercontent.com/assets/10705488/7441405/5a2cd9a6-f0bd-11e4-8b2a-c0dcbe85c0a4.png)
![1](https://cloud.githubusercontent.com/assets/10705488/7441406/5d6484e8-f0bd-11e4-9aaa-6c5191fe5ff2.png)

**Comment by Predator82Germany** [comment]
Update anything

**Comment by gonetz** [comment]
everything

**Comment by Mushman** [comment]
Your OpenGL DLL looks like it's dated July 2009. If that's accurate, then it may only be capable of OpenGL 3.1 at the moment. Have you updated your drivers at all? Here is the last release for the ATI Radeon HD 4000 series [for Windows 7 x64](http://support.amd.com/en-us/download/desktop/legacy?product=legacy2&os=Windows%207%20-%2064).

You'll probably need to post more information which might help us troubleshoot.

**Comment by gonetz** [comment]
Check for gliden64.log file. It is created if something goes wrong.

**Comment by Lithium64** [comment]
@Predator82Germany @gonetz I updated all the drivers and the black screen continues, the gliden64.log file wasn't created

**Comment by gonetz** [comment]
Hmm... Try to disable frame buffer emulation. It is the last idea.

**Comment by Predator82Germany** [comment]
All runtimes (DirectX 9, Visual C+)?

**Comment by Mushman** [comment]
You need to provide enough details for us to reproduce or understand your problem. Have you tried altering every setting? Does it happen on both versions of the plugin? Do you have access to any other computers to test? Are other plugins affected? The issue tracker isn't intended to provide support, rather, it is for reporting exactly how software fails so that it can be fixed.

**Comment by Lithium64** [comment]
@Mushman my video card is one ATI XFX HD 4650 1GB and the last driver was released in 2013, because it is a legacy card, but this card has OpenGL 3.3 support, if the plugin not works it is a issue on the driver or in the plugin, if it is a issue on the driver one workaround is needed since this card hasn't supported anymore by the manufacturer

@Predator82Germany yes

@gonetz I tried that too

**Comment by Mushman** [comment]
Plugin displays graphics when tested on an NVidia Geforce 330M which only has OpenGL 3.3 support using Mupen64Plus running Jet Force Gemini. This is not an OpenGL version issue, and is unlikely to be an AMD platform issue because the plugin was developed on that hardware.

**Comment by purplemarshmallow** [comment]
what happens if someone uses old drivers from 2013 on a newer AMD card?
maybe this can help to find the problem

**Comment by Lithium64** [comment]
@Mushman The latest driver for this card is from 2013 this driver could have a unfixed bug, the AMD OpenGL drivers has a bad history.

**Comment by Lithium64** [comment]
@gonetz I tested with mupen64plus 2.5 and appeared this error

![teste](https://cloud.githubusercontent.com/assets/10705488/7448905/83ae80a4-f1fd-11e4-9ae4-4baae0da3162.png)

**Comment by Mushman** [comment]
Okay, I found someone else with your issue [over on the Project64 forums](http://forum.pj64-emu.com/showthread.php?t=5027). Interestingly, their ATI Radeon HD 7850 runs OpenGL 4.4 and is using the latest drivers. Perhaps it is an AMD problem, but there has to be another factor causing it, as it's not happening on all that manufacturer's hardware.

**Comment by gonetz** [comment]
OpenGL Error on start of mupen64plus is known issue. Something wrong happens inside emulator's initialization code. I just check that error on start to not miss my own errors.

**Comment by krnlyng** [comment]
i have the black screen too on linux but only if i use the open source driver, with catalyst it works fine, what information can i provide to fix this?

**Comment by krnlyng** [comment]
(radeon HD 5770)

**Comment by krnlyng** [comment]
it shows many opengl errors during runtime: https://bpaste.net/show/3cfb5ae3be31

**Comment by krnlyng** [comment]
i have tried setting all options to False (if not already set to False) one after another but the screen remains black

**Comment by Lithium64** [comment]
@gonetz I tried to debug the plugin to find out what was happening through the GLinterceptor 1.2 debugging tool and found only this error

GL Intercept Log. Version : 1.32    Compile Date: Feb 23 2015    Run on: Fri May 08 17:23:25 2015

InterceptShaderGLSL::DeleteObjectPost - Unable to find shader ID 0

Log End.

**Comment by singleentry** [comment]
I have an ATI Radeon HD 5870 card on my laptop and I had this same black screen problem word for word, so I created an account to post my fix.

The existing drivers I had installed were up to date, but suspiciously my openGL compatibility ended at 3.3 same as you. No matter what I tried to do (disable->re-enable->update, or uninstall->reinstall->update), it seems my driver suite never saw patches past 3.3.

So I trashed the existing drivers I had and downloaded a completely different suite, the AMD Catalyst drivers, which also supported my hardware. Confirm they are compatible for you with the auto-detect feature http://support.amd.com/en-us/download

After the install and update, I'm at full OpenGL 4.4 compatibility now and the plugin works perfectly!

Small warning though! The AMD Catalyst suite also piggy-back installed audio drivers on top of the ones I already had. So two sets of drivers tried to communicate with the same device at once and killed the sound when I was plugged to an HDMI cable (only hdmi, speakers were fine). If this happens to you too, this fix worked for me: http://www.eightforums.com/sound-audio/21357-fix-amd-hdmi-no-sound-output-monitor-speakers.html

I hope this helps!
